client/pyroclient/client.py

```diff
@@ -123,52 +187,186 @@ def send_alert_from_device(self, lat: float, lon: float, event_id: int, media_id
+            >>> api_client = client.Client("http://pyronear-api.herokuapp.com", "MY_LOGIN", "MY_PWD")
+            >>> response = api_client.get_my_devices()
+
+        Returns:
+            HTTP response containing the list of owned devices
+        """
         return requests.get(self.routes["get-my-devices"], headers=self.headers)
 
     def get_sites(self) -> Response:
-        """Get all the existing sites in the DB"""
+        """Get all the existing sites in the DB
+
+        Example::
+            >>> from pyroclient import client
+            >>> api_client = client.Client("http://pyronear-api.herokuapp.com", "MY_LOGIN", "MY_PWD")
+            >>> response = api_client.get_sites()
+
+        Returns:
+            HTTP response containing the list of sites
+        """
         return requests.get(self.routes["get-sites"], headers=self.headers)
 
     def get_all_alerts(self) -> Response:
-        """Get all the existing alerts in the DB"""
+        """Get all the existing alerts in the DB
+
+        Example::
+            >>> from pyroclient import client
+            >>> api_client = client.Client("http://pyronear-api.herokuapp.com", "MY_LOGIN", "MY_PWD")
+            >>> response = api_client.get_all_alerts()
+
+        Returns:
+            HTTP response containing the list of all alerts
+        """
         return requests.get(self.routes["get-alerts"], headers=self.headers)
 
     def get_ongoing_alerts(self) -> Response:
-        """Get all the existing alerts in the DB that have the status 'start'"""
+        """Get all the existing alerts in the DB that have the status 'start'
+
+        Example::
+            >>> from pyroclient import client
+            >>> api_client = client.Client("http://pyronear-api.herokuapp.com", "MY_LOGIN", "MY_PWD")
+            >>> response = api_client.get_ongoing_alerts()
+
+        Returns:
+            HTTP response containing the list of all ongoing alerts
+        """
+
         return requests.get(self.routes["get-ongoing-alerts"], headers=self.headers)
 
     def get_unacknowledged_alerts(self) -> Response:
-        """Get all the existing alerts in the DB that have the field "is_acknowledged" set to `False`"""
+        """Get all the existing alerts in the DB that have the field "is_acknowledged" set to `False`
+
+        Example::
+            >>> from pyroclient import client
+            >>> api_client = client.Client("http://pyronear-api.herokuapp.com", "MY_LOGIN", "MY_PWD")
+            >>> response = api_client.get_unacknowledged_alerts()
+
+        Returns:
+            HTTP response containing the list of all alerts that haven't been acknowledged
+        """
         return requests.get(self.routes["get-unacknowledged-alerts"], headers=self.headers)
 
     def acknowledge_alert(self, alert_id: int) -> Response:
-        """Switch the `is_acknowledged`field value of the alert to `True`"""
+        """Switch the `is_acknowledged`field value of the alert to `True`
```

Comment:
missing space, while we are at it
```suggestion
        """Switch the `is_acknowledged` field value of the alert to `True`
```